cypress/integration/PressReleases/index.js

```diff
@@ -0,0 +1,63 @@
+/// <reference types="Cypress" />
+import { Given, And, Then } from 'cypress-cucumber-preprocessor/steps';
+
+And('the 2019 release doesnt have a link', () => {
+    cy.get('p').should('contain.text', '2019').should('not.have','a');
```

Comment:
2019 should be variable passed in, not hardcoded value - surround it with quotes in the feature file and make it a string, or pass it as int instead ( whatever works)

Reply:
updated this.